gameFunctions.js

```diff
@@ -0,0 +1,288 @@
+        let count = 0;
+        players.forEach(player => {
+            request.post({
+                url: 'https://slack.com/api/conversations.kick',
+                json: true,
+                headers: {
+                    Authorization: 'Bearer ' + process.env.SLACK_AUTH_TOKEN
+                },
+                body: {
+                    channel: channelId,
+                    user: player.id
+                }
+            }, (err, res, body) => {
+                console.log(body);
+                if (err) {
+                    console.log(err);
+                }
+
+                if (body.ok) {
+                    count++;
+                    if (count >= players.length) {
+                        resolve();
+                    }
+                } else if (body.error == 'cant_kick_self') {
+                    request.post({
+                        url: 'https://slack.com/api/conversations.leave',
+                        json: true,
+                        headers: {
+                            Authorization: 'Bearer ' + process.env.SLACK_AUTH_TOKEN
+                        },
+                        body: {
+                            channel: channelId
+                        }
+                    }, (err, res, body) => {
+                        console.log(body);
+                        if (err) {
+                            console.log(err);
+                        }
+
+                        if (body.ok) {
+                            count++;
+                            if (count >= players.length) {
+                                resolve();
+                            }
+                        }
+                    });
+                }
+            });
+        });
+    });
+    return response;
+}
+
+const createGroup = (players, channelId) => {
+    players.forEach(player => {
+        request.post({
+            url: 'https://slack.com/api/conversations.invite',
+            json: true,
+            headers: {
+                Authorization: 'Bearer ' + process.env.SLACK_AUTH_TOKEN
+            },
+            body: {
+                channel: channelId,
+                users: player.id
+            }
+        }, (err, res, body) => {
+            console.log(body);
+            if (err) {
+                console.log(err);
+            }
+        });
+    });
+
+}
+
+const getRoleText = (role) => {
+    switch (role) {
+        case 'seer':
+            return 'You are the seer. Each night you get to view one player. You win when all the BGs are dead.';
+        case 'doc':
+            return 'You are the doctor. Each night you get to save one player from a night death. You win when all the BGs are dead.';
+        case 'bg':
+            return 'You are Bad Guy. You win when there are the same number (or more) BGs left alive than GGs.';
+        default:
+            return 'You are just a plane jane villager. You win when all the BGs are dead.';
+    }
```

Comment:
These probably should be parameterized strings in the long-term or the text should be in the Database table for roles.

Reply:
Will get moved to DB with roles